doc/cookbook/source/examples/regression/kernel_ridge_regression.rst

```diff
@@ -0,0 +1,55 @@
+=======================
+Kernel Ridge Regression
+=======================
+
+Kernel ridge regression is a kernel-based regularized form of regression which learns a function in the space induced by the respective kernel and the data by minimizing a squared error loss with :math:`L_2` regularization. For linear ridge regression, it boils down to solving a linear system:
+
+.. math::
+    {\bf w} = \left({\bf X}^T{\bf X}+\tau{\bf I}\right)^{-1}{\bf y}
+
+where :math:`{\bf X}` is the feature matrix, :math:`{\bf y}` is the vector of labels and :math:`\tau>0` scales the regularization term.
+
+In case of kernels, a non-linear mapping :math:`\Phi : x \rightarrow \Phi(x) \in \mathcal F` is used. By using kernel trick :math:`K({\bf x_i, x_j})=\Phi({\bf x_i})\Phi({\bf x_j})` the following solution can be obtained:
```

Comment:
Actually, please remove Phi here completely. We dont need it in an API example, it doesnt matter for that